### ISSUE TITLE
Is mage.exe (from .NET Fx 4.7 SDK) redistributable?

### ISSUE THREAD

**Issue by chrischu** [microsoft/dotnet#440]
I'm not sure this is the right place to ask (if not it would be great if someone could point me in the right direction), but I was wondering if it is allowed to redistribute (i.e. deploy it with and call it from my application) mage.exe from the .NET Framework 4.7 SDK.

According to various licenses I found only files mentioned in the "redist.txt" can be redistributed freely, unfortunately I was unable to find any "redist.txt" file for this version of the SDK. Does this mean it is no longer allowed to redistribute mage.exe (I found some "redist.txt" files from other products (e.g. VS2010) that explicitly listed mage.exe as redistributable)?

Thanks in advance!

**Comment by richlander** [comment]
Interesting question. Can you share more about what you want to do? If you want to do that privately, you can mail me at rlander@ms.

**Comment by Petermarcu** [comment]
Can this be closed now?

**Comment by chrischu** [comment]
Yes.

**Comment by jnl002** [comment]
What was the outcome of this? Was it determined that Mage.exe is still distributable?

**Comment by richlander** [comment]
> deploy it with and call it from my application

Does this mean to your customers? Knowing that will help define the scope.

**Comment by chrischu** [comment]
Yes, we would ship or software in Docker containers that run both in our own Azure subscription and in our customers own in-premises environments.

**Comment by chrischu** [comment]
@richlander Any update on this?

**Comment by chrischu** [comment]
@richlander Any news?
